src/window.js

```diff
@@ -29,14 +29,13 @@ export class Window {
     open(e) {
         this.e = document.createElement('div')
         this.e.className = "window"
-        this.e.id = `tab-${this.gate.id}.${this.id}`
+        this.e.id = `${this.gate.name}.${this.id+1}`
```

Comment:
Not sure why this change is here.

Reply:
also, I think ids have strict rules for the chars their support while names don't. i.e. a name can have a space in it and ids can't